src/App.js

```diff
@@ -0,0 +1,20 @@
+import React, { Fragment, Component } from 'react'
+import Welcome from './components/welcome.js'
```

Comment:
Files that contains components should be capitalize: `Welcome.js` instead of `welcome.js`